tests/test_tally.py

```diff
@@ -436,6 +436,63 @@ def test_mcnp6_tally_vol_proton_volume_set():
                  "F14:h 12\n"+\
                  "SD14 100.000000\n",tally.mcnp(1,"mcnp6"))
 
+def test_mcnp_mesh_tally_xyz():
+    particle = "Neutron"
+    geometry = "Cartesian"
+    origin = [1,2,3]
+    i = [5, 10, 20, 25]
+    j = [2, 12]
+    k = [45]
+    i_ints = [1, 2, 3, 1]
+    j_ints = [1]
+    k_ints = [1]
+    e = [0, 10, 100]
+    e_ints = [1, 1, 2]
+    tal_name = "Mesh Tally XYZ Proton"
```

Comment:
for the sake of confusion downstream - maybe change either the title to neutron or the particle to proton